drivers/sensor/akm09918c/akm09918c.h

```diff
@@ -74,4 +74,18 @@ static inline void akm09918c_reg_to_hz(uint8_t reg, struct sensor_value *val)
 	}
 }
 
+/*
+ * RTIO types
+ */
+
+struct akm09918c_decoder_header {
+	uint64_t timestamp;
+	uint8_t is_fifo: 1;
```

Comment:
The AKM sensor doesn't have a FIFO, you can remove this since it'll always be `0`